ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-commands.sh

```diff
@@ -157,6 +157,10 @@ if [ ${SECURE_BOOT_ENABLED} = "true" ]; then
   sed -i '/guest_id/a\  firmware         = "efi"\n  efi_secure_boot_enabled = "true"' ./vm/main.tf
 fi
 
+if [ ${VMS_ENCRYPTED} = "true" ]; then
+  sed -i '/guest_id/a\ storage_policy_id = "b1263970-8662-69e2-adc6-fa8ae01abecc"' ./vm/main.tf
```

Comment:
The `storage_policy_id` need to be created in advanced in the vsphere deployment, right?

Reply:
It is bundled encryption policy with vcenter, pls refer to https://github.com/openshift/release/pull/36677#discussion_r1123996610

Reply:
Here the storage policyID use a preset vsphere encryption policy of the vsphere and it's also a recommended config. Maybe we could use the policy name gets its id if it's available by `govc` commandline, it will expand for other vsphere envs using the step. The preset policy name is `Management Storage policy - Encryption`.

Reply:
If possible, though it is a bundled encryption policy with vcenter, I still suggest to prompt it as ENV var for the step. So that allow users to switch over the id easily when needed.

Reply:
Maybe we can replace `VMS_ENCRYPTED` with `STORAGE_POLICY_ID` with an empty default value.

Reply:
@Phaow Is it OK for you that we use STORAGE_POLICY_ID to expand for other vsphere envs?

Reply:
@Phaow 's idea is good too. But not sure if the name `Management Storage policy - Encryption` never get changed on different vsphere deployment.

Reply:
I'm ok. For storageClass configuration it support using a different encryption policy. I used ENV var `REQUIRED_ENCRYPTION_POLICY` in my chain.
https://github.com/openshift/release/pull/37476/files#diff-7cb784c44d0975c126d36bef87b50cfd7d14095787f9776d8b10c4a1b17de58aR24

Reply:
Or we can keep `VMS_ENCRYPTED` env, then run `govc` command to confirm `Management Storage policy - Encryption` existing or not, if not, then exit as failure, if yes, use it. The pre-check would help us reduced some deubugging time.

Reply:
Good option,did a quick check for the preset  vsphere encryption policy `Management Storage policy - Encryption` should be always exist even if the administator couldn't delete it only could edit it.

Reply:
btw, can I understand the vm encrypt is some similar things to disk encryption on other cloud platform?

Reply:
It has some similar things with disk encryption on other cloud platform. But currently vsphere more depends on an exteral kms key managemant server(key provider) do the encryption key management and AFAIK we couldn't specified 1 kms key to encryption the disks like other platform use a keyID it only could set the policy and a default kms key management server.
https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-8D7D09AC-8579-4A33-9449-8E8BA49A3003.html
https://kb.vmware.com/s/article/2147566